extensions/wikia/Insights/Insights.i18n.php

```diff
@@ -60,7 +60,8 @@
 
 	'insights-module-see-more' => 'See more insights',
 
-	'insights-label-altaction-infoboxes' => 'Convert!',
+	'insights-altaction-convert' => 'Convert!',
+	'insights-altaction-seedraft' => 'View the draft',
```

Comment:
Needs some QQQ, and `insights-label-altaction-infoboxes` should be removed form the QQQ docs.